src/main/java/io/github/thebusybiscuit/slimefun4/implementation/setup/SlimefunItemSetup.java

```diff
@@ -1106,6 +1107,10 @@ public static void setup(@Nonnull SlimefunPlugin plugin) {
         new ItemStack[] {SlimefunItems.GOLD_24K, null, SlimefunItems.GOLD_24K, new ItemStack(Material.LEATHER), SlimefunItems.GILDED_BACKPACK, new ItemStack(Material.LEATHER), SlimefunItems.GOLD_24K, null, SlimefunItems.GOLD_24K})
         .register(plugin);
 
+        new ContainmentBackpack(27, categories.usefulItems, SlimefunItems.CONTAINMENT_BACKPACK, RecipeType.ENHANCED_CRAFTING_TABLE,
+        new ItemStack[] {SlimefunItems.LEAD_INGOT, SlimefunItems.COOLING_UNIT, SlimefunItems.LEAD_INGOT, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.BACKPACK_LARGE, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH, SlimefunItems.REINFORCED_CLOTH})
```

Comment:
Why the Cooling Unit exactly?

Reply:
"Keep the radioactive stuff to a cool temp"
As i said recipe, name, and texture are all just from the top of my head